torch/_inductor/lowering.py

```diff
@@ -1920,9 +1916,11 @@ def unsupported_input_tensor(t: torch.Tensor, parent=None, node=None):
     return False
 
 
-def unsupported_output_tensor(t: torch.Tensor, parent=None, node=None):
+def unsupported_output_tensor(t: torch.Tensor, node=None):
     "Do not support writing tensor but can read from it"
-    if unsupported_input_tensor(t, parent):
+    if node is not None and node.target is aten.view.dtype and t.dtype.is_complex:
```

Comment:
I would love if this function was flipped to `supported_output_tensor` this double negatie is annoying